src/client/components/MealsList/MealsList.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState, useEffect } from 'react';
+import "./MealsList.css";
+import Meal from './Meal/Meal';
+
+function MealsList() {
+    const [meals, setMeals] = useState([]);
+
+    useEffect(() => {
+        fetch('http://localhost:3005/all-meals')
+            .then(response => response.json())
+            .then(data => setMeals(data))
+            .catch(error => console.error(error));
+    }, [meals]);
```

Comment:
Should the data be fetched every time the meals state change, or just on component mount []..?